test/test_joined_list.rb

```diff
@@ -0,0 +1,64 @@
+require 'helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.